kcidb/mq/__init__.py

```diff
@@ -15,8 +14,11 @@
 from google.api_core.exceptions import DeadlineExceeded
 import kcidb.io as io
 import kcidb.orm
-from kcidb import misc
+from kcidb.argparse import output_add_args
+from kcidb.orm.argparse import PatternHelpAction
 from kcidb.misc import LIGHT_ASSERTS
+from kcidb import misc
+from .argparse import SubscriberArgumentParser, PublisherArgumentParser
```

Comment:
Could we do `from kcidb.mq import argparse` instead, and use `argparse.SubscriberArgumentParser` and so on, instead?